<launch>
  <group ns="rero_ros">
    <!-- set /speech_recognition/ -->
    <param name="core_host" value="0.0.0.0" />
    <param name="core_port" value="50052" type="str" />
    <param name="topic_name" value="/rero_ros/recognized_text" />

    <node pkg="rero_ros" name="speech_recognition" type="speech_recognition" output="screen">
    </node>
  </group>
</launch>